docs/markdown/Reference-manual.md

```diff
@@ -1798,6 +1798,23 @@ an external dependency with the following methods:
 
  - `version()` is the version number as a string, for example `1.2.8`
 
+ - `partial_dependency(compile_args : false, link_args : false, links : false,
+   includes : false, source : false)` (*added 0.46.0) returns a new dependency
+   object with the same name, version, found status, type name, and methods as
+   the object that called it. This new object will only inherit other
+   attributes from it's parent as controlled by keyword arguments.
+   Additionally, any dependency objects that the parent has well generate a
+   partial_dependency with the same arguments and attach that to the new
+   object.
```

Comment:
This sentence seems a bit confusing. Maybe it would be better to explain this in a different way. Something like whether the dependencies of this dependency are used in the same way or whether they are also subset in the same way. That is, if you have a dependency with a subdependency that has a library, does `main_dep.partial_dependency(compile_args:true)` have the subdep's library or not.